src/controlpotmeter.h

```diff
@@ -74,6 +74,7 @@ class ControlPotmeter : public ControlObject {
     Q_OBJECT
   public:
     ControlPotmeter(ConfigKey key, double dMinValue = 0.0, double dMaxValue = 1.0,
+                    double dNeutralValue = 0.0,
```

Comment:
I've introduced PotmeterParameters class and a few more, but I get an error from the linker. I'm pretty sure it's some stupid mistake I can't see right now. Could you please take a look? Any further opinion on the new code is also welcome.